gcn_email_service/helpers.py

```diff
@@ -0,0 +1,21 @@
+import threading
+
+
+def periodic_task(interval, times=-1):
+    def outer_wrap(function):
+        def wrap(*args, **kwargs):
+            stop = threading.Event()
+
+            def inner_wrap():
+                i = 0
+                while i != times and not stop.is_set():
+                    stop.wait(interval)
+                    function(*args, **kwargs)
+                    i += 1
+
+            t = threading.Timer(0, inner_wrap)
+            t.daemon = True
```

Comment:
No need for a Timer here, since the initial interval is 0:

```suggestion
            t = threading.Thread(target=inner_wrap, daemon=True)
```